simplq/src/components/pages/Admin/ShareQueue.jsx

```diff
@@ -1,44 +1,23 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import FileCopyIcon from '@material-ui/icons/FileCopy';
-import { ShareButton } from '../../common/Button/Button.stories';
+import { useDispatch } from 'react-redux';
+import StandardButton from '../../common/Button';
+import { setInfoPopupMessage } from '../../../store/appSlice';
 
 const ShareQueue = (props) => {
-  const [clicked, setClicked] = useState(false);
   const shareUrl = `Hi! Use ${window.location.origin}/j/${props.queueName} to join my queue and get live updates.`;
-
-  useEffect(() => {
-    // https://www.alexhughes.dev/blog/setTimeout-with-hooks/
-    if (!clicked) {
-      return () => {};
-    }
-    // changes back to old text and color after 5 seconds
-    const timerId = setTimeout(() => setClicked(false), 5000);
-    return () => clearTimeout(timerId);
-  }, [clicked]);
+  const dispatch = useDispatch();
 
   const handleShareButtonClick = () => {
-    setClicked(true);
+    dispatch(setInfoPopupMessage('Copied to clipboard'));
   };
```

Comment:
We could change this to 'Copied queue link to clipboard' or 'You can now share queue details from the clipboard'

Reply:
Copied queue link to clipboard 👍